File: price-map/apps/scraper/scraper/entities/product_shop_match.py
class ProductShopMatch:
  """ Сопоставление товара и магазина (от товара - id, от магазина - название)
  """

  def __init__(self, product_id: str, shop_name: str) -> None:
    self.product_id: str = product_id
    """ Id товара """
    self.shop_name: str = shop_name
    """ Название магазина """

File: price-map/apps/overpass-requester/models/ProductShopMatch.cs
namespace Models;

class ProductShopMatch {
  private string ProductId;
  private string ShopName;
  public string product_id { get { return this.ProductId; } set { this.ProductId = value; } }
  public string shopName { get { return this.ShopName; } set { this.ShopName = value; } }
  public ProductShopMatch() {}
}